jetty-home/src/main/resources/bin/jetty.sh

```diff
@@ -430,7 +433,7 @@ case "`uname`" in
 CYGWIN*) JETTY_START="`cygpath -w $JETTY_START`";;
 esac
 
-RUN_ARGS=(${JAVA_OPTIONS[@]} -jar "$JETTY_START" ${JETTY_ARGS[*]})
+RUN_ARGS=$("$JAVA" ${JAVA_OPTIONS[@]} -jar "$JETTY_START" --dry-run=opts,path,main,args ${JETTY_ARGS[*]})
 RUN_CMD=("$JAVA" ${RUN_ARGS[@]})
```

Comment:
If I define `JAVA_OPTIONS="--show-version"` then `RUN_ARGS` is set to the wrong string, which fails starting.

I think this is more correct:

```suggestion
RUN_ARGS=$("$JAVA" -jar "$JETTY_START" --dry-run=opts,path,main,args ${JETTY_ARGS[*]})
RUN_CMD=("$JAVA" ${JAVA_OPTIONS[@]} ${RUN_ARGS[@]})
```

Reply:
@sbordet Good point, but I think the options need to end up in RUN_ARGS as they are sometimes used separately. We also need to protect again any options from module files that might make the output multiline.